cli/internal/helm/helm_test.go

```diff
@@ -107,3 +119,148 @@ func TestMergeMaps(t *testing.T) {
 		})
 	}
 }
+
+func TestHelmApply(t *testing.T) {
```

Comment:
It's helpful to use outdated versions (e.g. 1.99.12) where possible. During release preparation it sometimes happens that you have to look for todos related to a specific version. In those cases, the unittests create unnecessary noise.